packages/browser-wallet/package.json

```
@@ -1,7 +1,7 @@
 {
     "name": "@concordium/browser-wallet",
     "private": true,
-    "version": "2.4.0",
+    "version": "2.4.1",
```

I'd argue this is a minor release

```suggestion
    "version": "2.5.0",
```